rplugin/python3/deoplete/sources/deoplete_jedi/utils.py

```diff
@@ -50,6 +50,29 @@ def module_search(module, paths):
     return ''
 
 
+def rplugin_runtime_paths(context):
+    """Adds Neovim runtime paths.
+
+    Additional paths are added only if a `rplugin/python*` exists.
+    """
+    paths = []
+
+    if context and 'cwd' in context:
+        cwd = context.get('cwd')
+        rplugins = ('rplugin/pythonx',
+                    'rplugin/python{}'.format(sys.version_info[0]))
+        if any([os.path.exists(os.path.join(cwd, x))
+                for x in rplugins]):
+            for rtp in context.get('runtimepath', '').split(','):
+                if not rtp:
+                    continue
+                for r in rplugins:
+                    p = os.path.join(rtp, r)
+                    if os.path.exists(p):
```

Comment:
I guess with a map and filter.  I don't like doing that when prototyping because I really dislike deconstructing them if they have to increase in complexity.  I also have code I haven't pushed yet 😜